### PR TITLE
Dac6 1050

### PR DESCRIPTION
**Pull request by mbiczak**

### CODE REVIEW

File: app/pages/UTRPage.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+package pages
+
+import play.api.libs.json.JsPath
+
+case object UTRPage extends QuestionPage[String] {
+
+  override def path: JsPath = JsPath \ toString
+
+  override def toString: String = "uTR"
```
Comment (shaileshp0110) [inline app/pages/UTRPage.scala:25]: Why  this is lower case u

File: conf/views/utr.njk [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+              classes: "govuk-label--l text-overflow"
+            },
+            id: "value",
+            name: "value",
+            hint: {
+                        text: messages('utr.hint', taxType)
+                      },
+            value: form.value.value,
+            errorMessage: form.value.error
+          }) }}
```
Comment (shaileshp0110) [inline conf/views/utr.njk:50]: remove the commented code

File: app/controllers/BusinessNameController.scala [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+  val unincorporatedHeading = "name of your organisation"
+  val unincorporatedHint    = "This is the name on your governing document."
+
+  private def pageHeadingAndHint(businessType: BusinessType): (String, String) =
+    businessType match {
+      case LimitedPartnership | LimitedCompany => (llpHeading, llpHint)
+      case Partnership                         => (partnershipHint, partnershipHint)
+      case UnincorporatedAssociation           => (unincorporatedHeading, unincorporatedHint)
+    }
```
Comment (soljeggers) [inline app/controllers/BusinessNameController.scala:72]: Do these needs to be vals? Could we keep content in the messages file instead?

File: app/controllers/UTRController.scala [hunk 4]
```diff
@@ -55,15 +53,13 @@ class UTRController @Inject() (
     with I18nSupport
     with NunjucksSupport {
 
-  private val form = formProvider()
-  private val ct   = "Corporation Tax"
-  private val sa   = "Self Assessment"
+  private val ct = "Corporation Tax"
```
Comment (iservport) [inline app/controllers/UTRController.scala:56]: What about replacing hard-coded strings with message keys?

File: app/forms/mappings/Formatters.scala [hunk 5]
```diff
@@ -122,14 +122,22 @@ trait Formatters extends Transforms {
         baseFormatter.unbind(key, value.toString)
     }
 
-  private[mappings] def stringTrimFormatter(errorKey: String): Formatter[String] = new Formatter[String] {
+  private[mappings] def stringTrimFormatter(errorKey: String, msgArg: String = ""): Formatter[String] = new Formatter[String] {
```
Comment (iservport) [inline app/forms/mappings/Formatters.scala:125]: 👍

File: conf/views/soleName.njk [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+          {{ messages("soleName.heading") }}
+        </h1>
+
+          {{ govukInput({
+            label: {
+              text: messages('soleName.firstName')
+            },
+            id: "firstName",
+            name: "firstName",
+            autocomplete: "given-name",
```
Comment (iservport) [inline conf/views/soleName.njk:46]: 👍

**Review by iservport — APPROVED** [review]
LGTM

**Review by iservport — APPROVED** [review]
LGTM

**Pull request merged**